assets/css/main.scss

```diff
@@ -85,9 +85,26 @@ h1, h2, h3, h4, h5, h6 {
   position: relative;
 }
 
-
 .page__content {
   h1, h2, h3, h4, h5, h6 {
     margin: 0.5em 0;
   }
 }
+
+.figure-right-aligned,
+.figure-left-aligned {
```

Comment:
Can we make this `figure.align-right` and `figure.align-left`? Otherwise the HTML looks repetitive:

```
<figure class="figure-right-aligned">
...
```